src/index.js

```diff
@@ -9,7 +9,7 @@ const NEAR_NET_VERSION_TEST = '98';
 const utils = require('./utils');
 const nearToEth = require('./near_to_eth_objects');
 
-const GAS_AMOUNT = new BN('1000000000000000'); // 10e14.
+const GAS_AMOUNT = new BN('200000000000000');
```

Comment:
What was the error for this one? We've had to change the `GAS_AMOUNT` several times lol.

Reply:
```
InvalidTxError: {}
          at Object.parseRpcError (/Users/emilywilliams/code/summa/near/near-web3-provider/node_modules/near-api-js/lib/utils/rpc_errors.js:38:19)
          at JsonRpcProvider.sendJsonRpc (/Users/emilywilliams/code/summa/near/near-web3-provider/node_modules/near-api-js/lib/providers/json-rpc-provider.js:120:40)
          at processTicksAndRejections (internal/process/task_queues.js:97:5)
          at Account.signAndSendTransaction (/Users/emilywilliams/code/summa/near/near-web3-provider/node_modules/near-api-js/lib/account.js:97:22)
          at NearProvider.routeEthSendTransaction (/Users/emilywilliams/code/summa/near/near-web3-provider/src/index.js:714:27) {
        type: 'InvalidTxError'
      }
```

Reply:
not very helpful --^